cypress/integration/1-getting-started/Local tests/ONE/ONE_1_Webpage_tour_test.spec.js

```diff
@@ -0,0 +1,22 @@
+/// <reference types="cypress" />
+const baseURL = Cypress.env('baseURL')
+// Before running this test first run the Server test 2 in order to create a user so you can log in.
+it('should be able to visit the login page', () => {
+    cy.visit(baseURL + '/') // Localhost in the env file should be changed to reflect the ONE host
+})
+
+it('should be able to log in successfully', () => {
+    cy.get('[data-test-id="input-email"]').type("cypress@test.com") // Here write your email address for your account
+    cy.get('[data-test-id="input-password"]').type("cypress123") // Write the password for your account
+    cy.get('[data-test-id="button-login-and-remember"]').click() // Here we click on the log in and remember me button
+    cy.visit(baseURL + '/') // Visiting main page of ONE
+})
```

Comment:
Again you're in the folder 'One', no need to include the word 'one' in the file name